ios/FluentUI/Badge Field/BadgeViewTokenSet.swift

```diff
@@ -0,0 +1,199 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+/// Pre-defined styles of the Badge.
+@objc public enum MSFBadgeViewStyle: Int, CaseIterable {
```

Comment:
i think i saw similar comment before but is there a reason we have to change the enum signature?     @objc(MSFBadgeViewStyle)
    public enum Style: Int {

Reply:
I had seen this done in other token files so I thought this was standard practice. I'll update.